.github/workflows/build-crate-and-npm.yml

```diff
@@ -54,122 +54,13 @@ jobs:
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/chia-bls
-        cargo publish
-
-    - name: publish to crates.io if tagged (clvm-derive)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/clvm-derive
-        cargo publish
-
-    - name: publish to crates.io if tagged (clvm-traits)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/clvm-traits
-        cargo publish
-
-    - name: publish to crates.io if tagged (clvm-utils)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/clvm-utils
-        cargo publish
-
-    - name: publish to crates.io if tagged (chia-protocol)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/chia-protocol
-        cargo publish
-
-    - name: publish to crates.io if tagged (chia-puzzles)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/chia-puzzles
-        cargo publish
-
-    - name: publish to crates.io if tagged (chia-client)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/chia-client
-        cargo publish
-
-    - name: publish to crates.io if tagged (chia-ssl)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/chia-ssl
-        cargo publish
-
-    - name: publish to crates.io if tagged (chia-consensus)
-      continue-on-error: true
-      if: startsWith(github.event.ref, 'refs/tags')
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
-      run: |
-        cd crates/chia-consensus
-        cargo publish
-
-    - name: publish to crates.io if tagged (chia)
-      continue-on-error: true
+    - name: publish to crates.io if tagged
       if: startsWith(github.event.ref, 'refs/tags')
       env:
         CARGO_REGISTRY_TOKEN: ${{ secrets.cargo_registry_token }}
       run: |
-        cargo publish
+        cargo install cargo-workspaces --locked
+        cargo ws publish --publish-as-is --allow-dirty
```

Comment:
I seem to recall that `--allow-dirty` is risky. Does it not include the dirty files in the package?